injected/integration-test/duck-ai-data-clearing.spec.js

```diff
@@ -0,0 +1,183 @@
+import { test, expect } from '@playwright/test';
+import { ResultsCollector } from './page-objects/results-collector.js';
+
+const HTML = '/duck-ai-data-clearing/index.html';
+const CONFIG = './integration-test/test-pages/duck-ai-data-clearing/config/enabled.json';
+
+test('duck-ai-data-clearing feature clears localStorage and IndexedDB', async ({ page }, testInfo) => {
+    const collector = ResultsCollector.create(page, testInfo.project.use);
```

Comment:
I 100% wouldn't change this now but I myself wouldn't have integration tested this given this amount of boilerplate (sorry).

For simpler stuff there's a less boilerplate direction [like this.](https://github.com/duckduckgo/content-scope-scripts/blob/6f948b2191a9ed88451ffd6c584cc047b1c3445d/injected/integration-test/pages.spec.js#L62-L69) but I don't think that'd work here.